### PR TITLE
Add staging variables for payments admin deployment

### PR DESCRIPTION
**Pull request by jphillips2121**

### CODE REVIEW

File: groups/stack/profiles/development-eu-west-2/staging/vars [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+state_prefix = "env:/staging"
+aws_profile = "staging-eu-west-2"
+
+cookie_domain = "companieshouse.gov.uk"
+cookie_name = "__SID"
+
+# Certificate for https access through ALB
+ssl_certificate_id = "arn:aws:acm:eu-west-2:250991044064:certificate/56b76dab-5728-4f83-a16d-e3dd59cd82c8"
+external_top_level_domain = "-staging-aws.companieshouse.gov.uk"
+internal_top_level_domain = "-internal-api-staging-aws.companieshouse.gov.uk"
```
Comment (macreman) [inline groups/stack/profiles/development-eu-west-2/staging/vars:14]: I think this one should be 

`internal_top_level_domain = ".staging.aws.internal"`

File: groups/stack/profiles/development-eu-west-2/staging/vars [hunk 2]
```diff
@@ -0,0 +1,38 @@
+aws_bucket = "staging-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-staging-terraform-state"
+environment = "staging"
+deploy_to = "staging"
+state_prefix = "env:/staging"
+aws_profile = "staging-eu-west-2"
+
+cookie_domain = "companieshouse.gov.uk"
```
Comment (macreman) [inline groups/stack/profiles/development-eu-west-2/staging/vars:8]: `cookie_domain = "company-information.service.gov.uk"`

File: groups/stack/profiles/development-eu-west-2/staging/vars [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ec2_key_pair_name = "chs-aws-staging"
+
+# shared configs
+log_level = "TRACE"
+
+# eric configs
+eric_cache_url = "staging-chs-elasticache-001.3psdof.0001.euw2.cache.amazonaws.com:6379"
+
+# api configs
+internal_api_url = "https://internal-api-staging-aws.companieshouse.gov.uk"
```
Comment (macreman) [inline groups/stack/profiles/development-eu-west-2/staging/vars:25]: pretty sure this and all of the configs below that end with `companieshouse.gov.uk` should end with `company-information.service.gov.uk` instead

File: groups/stack/profiles/development-eu-west-2/staging/vars [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+api_url = "https://api-staging-aws.companieshouse.gov.uk"
+
+# shared payments admin web configs (for payments admin web and in future regular public payments web)
+pay_api_url = "api-payments-staging-aws.companieshouse.gov.uk"
+pay_web_oauth2_redirect_uri = "https://staging-aws.companieshouse.gov.uk/oauth2/user/callback"
+pay_web_oauth2_token_uri = "https://account-staging-aws.companieshouse.gov.uk/oauth2/token"
+pay_web_cdn_host = "d309scb72axzzb.cloudfront.net"
```
Comment (macreman) [inline groups/stack/profiles/development-eu-west-2/staging/vars:32]: current cdn host in staging mesos golbal_env is 
`export CDN_HOST=d3s4qnqz7t874x.cloudfront.net # cdn.ch.gov.uk-3.22.170`
so unless theres a specific reason to have a different one (maybe this one is newer and not in global_env yet?) we should probably match this

**Review by macreman — CHANGES_REQUESTED** [review]

**Review by emead — APPROVED** [review]

**Pull request merged**